### PR TITLE
Introduce pre-release workflow to ship alpha versions of our packages

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
We want to develop with confidence and have a pre-release flow so we can QA our changes before publishing in npm.

## TODO
- [x] Create pre-release CI 
- [x] Explain how release and pre-release work on `RELEASE_PROCESS.md`
- [x] Create a `dev` branch

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Release process
+We publish Latitude packages in npmjs.com and GitHub. We use [semantic versioning](https://semver.org/) to manage the versioning of our packages.
+
+We have two types of publications. We publish to `latest` which is the stable
+versions 1and `next` which is the pre-release versions.
```
Comment (geclos) [inline RELEASE_PROCESS.md:5]: type 1and

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+We have a [release.yml](./.github/workflow/release.yml) CI workflow that
+publish to `latest` dist tag in npmjs.com (default) when changes are merged into `main` branch.
+To make a release, you need to:
+
+1. Start a branch from `main` and do the changes you want
+1. Create a changeset for your changes with `pnpm changeset add`.
+3. Open a pull request in GitHub pointing to `main`
+
+When the pull request is merged, the CI workflow will create a PR with the
+changesets. At some point someone with permissions will merge this branch into `main` and a release will be published.
```
Comment (geclos) [inline RELEASE_PROCESS.md:33]: very cool 🆗 

i think we could centralize all this info, together with info on how to do development, in a single CONTRIBUTING md, like evidence [does](https://github.com/evidence-dev/evidence/blob/next/CONTRIBUTING.md)

**Review by geclos — APPROVED** [review]

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (andresgutgon) [reply RELEASE_PROCESS.md:33]: Yes, I'll move it

**Comment by geclos** [comment]
@andresgutgon Closing this for now. I think we don't have the real need to do this atm, and we have our plate full anyways. Feel free to repon if you disagree.

File: .github/workflows/pre-release.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+name: Pre-Release
+
+on:
+  push:
+    branches:
+      - dev
```
[the remark this answers is not in the document]
Comment (andresgutgon) [reply .github/workflows/pre-release.yml:6]: I like more `canary` 🐦

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -122,10 +122,39 @@ Changesets ensure that package versions are updated correctly before releasing o
...
+
+Our [pre-release.yml](./.github/workflow/pre-release.yml) CI workflow automatically publishes to the `next` distribution tag on npmjs.com when changes are merged into the `next` branch.
+To make a pre-release, follow these steps:
+
+1. Switch to the `next` branch and run `git rebase main` to ensure it's up to date with the latest package versions.
+2. Branch off `next` for your changes.
+3. Create a changeset for your modifications using `pnpm changeset pre next`. The `next` tag specifies that this version will be published on npmjs.com.
+4. Open a pull request on GitHub targeting the `next` branch.
+
+Once the pull request is merged, the CI will generate a PR with the changesets. Eventually, someone with permissions will merge this PR into `dev`, triggering the publication of a pre-release.
```
Comment (andresgutgon) [inline CONTRIBUTING.md:147]: `next` no?

Comment (geclos) [reply CONTRIBUTING.md:147]: si

**Pull request merged**